packages/react-components/react-popover/src/common/mockUsePopoverContext.ts

```diff
@@ -15,6 +15,8 @@ export const mockPopoverContext = (options: Partial<PopoverContextValue> = {}) =
     arrowRef: { current: null },
     openOnContext: false,
     openOnHover: false,
+    mouseLeaveDelay: 500,
+    setOpenTimeoutRef: { current: 0 },
```

Comment:
I'd avoid putting this in context. It shouldn't be necessary. if you look at how `Menu` does ithttps://github.com/microsoft/fluentui/blob/fd41e5b15e3a2f08245cd4e4bbcc0f07a367ce3b/packages/react-components/react-menu/src/components/Menu/useMenu.tsx#L155-L174

Reply:
you can also avoid the delay in the context too

Reply:
Updated :)